### PR TITLE
Bug 1195657 - Focus on the leftmost card on initial

### PR DESCRIPTION
**Pull request by Fischer-L**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d38ca2e37a3785f3dc153ae5b2446c222969d6e4

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -164,6 +162,8 @@
                         that.onCardRemoved.bind(that, that.folderScrollable));
           }
         });
+
+        that.spatialNavigator.focus(that.spatialNavigator._collection[3]);
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: Use `that.cardScrollable` as arguments.

**Comment by rexboy7** [comment]
use:
`Bug 1195657 - [Stingray][Home] Focus on the leftmost card on initial`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6f66f2f3b41571a8f22e95cf83ef2c8d2c607794

**Pull request merged**